File: CloudMusic/src/main/java/com/ken/android/CloudMusic/Activity/View/OneThirdOfParentLayout.java
package com.ken.android.CloudMusic.Activity.View;

import android.content.Context;
import android.graphics.Canvas;
import android.util.AttributeSet;
import android.view.View;
import android.view.ViewGroup;

/**
 * Created by axnshy on 16/8/9.
 */
public class OneThirdOfParentLayout extends ViewGroup{
    public OneThirdOfParentLayout(Context context) {
        super(context);
    }

    public OneThirdOfParentLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public OneThirdOfParentLayout(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public OneThirdOfParentLayout(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }

    @Override
    protected void onLayout(boolean b, int i, int i1, int i2, int i3) {

    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, heightMeasureSpec);

    }
}

File: SmartPlay/src/main/java/com/luffy/smartplay/ui/view/OneThirdOfParentLayout.kt
package com.luffy.smartplay.ui.view

import android.view.ViewGroup
import android.content.*
import android.util.AttributeSet

/**
 * Created by axnshy on 16/8/9.
 */
class OneThirdOfParentLayout : ViewGroup {
    constructor(context: Context?) : super(context) {}
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {}
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    ) {
    }

    constructor(
        context: Context?,
        attrs: AttributeSet?,
        defStyleAttr: Int,
        defStyleRes: Int
    ) : super(context, attrs, defStyleAttr, defStyleRes) {
    }

    override fun onLayout(b: Boolean, i: Int, i1: Int, i2: Int, i3: Int) {}
    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
    }
}